src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -2372,11 +2376,8 @@ private boolean isFeedbackSessionViewableTo(
         }
```

Comment:
Although the SHPUiTest lets it pass, the logic test caught it so I don't need to add any. I haven't run the latest changes against the full test run yet.